cdap-explore/src/test/java/co/cask/cdap/explore/service/HiveExploreServiceTestRun.java

```diff
@@ -273,6 +276,26 @@ public void testHiveIntegration() throws Exception {
   }
 
   @Test
+  public void testQueriesCount() throws Exception {
+    Id.Namespace testNamespace1 = Id.Namespace.from("testQueriesCount");
+    exploreClient.addNamespace(testNamespace1).get();
```

Comment:
This may need to be updated now to use a combination of ``NamespaceClient`` and ``DatasetFramework`` to manage namespaces in explore tests. Look at how its done in ``BaseHiveExploreServiceTest`` in latest develop.